classes/author/Collector.inc.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * @file classes/author/Collector.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class author
+ *
+ * @brief A helper class to configure a Query Builder to get a collection of authors
+ */
+
+namespace APP\author;
+
+use APP\core\Application;
+use Illuminate\Database\Query\Builder;
+use Illuminate\Support\Facades\DB;
+
+class Collector extends \PKP\author\Collector
+{
+    /** @var int|null */
+    public $chapterId = null;
+
+    public function __construct(\PKP\author\DAO $dao)
+    {
+        $this->dao = $dao;
+    }
+
+    /**
+     * Limit results to submissions assigned to these series
+     */
+    public function filterByChapter(int $chapterId): self
+    {
+        $this->chapterId = $chapterId;
+        return $this;
+    }
```

Comment:
Let's make this an array, so `filterByChapterIds(array $chapterIds)`, then update the query below to use `->whereIn(`

Reply:
(`?array`, that is)

Reply:
Changed